### PR TITLE
SGDS-131 add accordion styles w existing selectors and custom scss

### PR DESCRIPTION
**Pull request by clukhei**
# Description

- Tweak accordion x paddings using selectors (from bootstrap 's 20px to 24px)
- removed accordion shadow box when open 
- accordion button font 700 and purple-500 when accordion is active 
- set accordion's line height to 32px 

Fixes # (issue number)

## Type of change

Please check on the checkbox for those that are relevant (put x between the brackets)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Code enhancement and update (non-breaking change)
- [ ] Security patch

## How Has This Been Tested?

Please describe or list the test cases that you ran to verify your changes, and provide instructions so we can reproduce them.

### CODE REVIEW

File: sgds/sass/sgds-theme/components/_accordion.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.sgds{
+    &.accordion{
+        line-height: 2rem;
```
Comment (rav3) [inline sgds/sass/sgds-theme/components/_accordion.scss:3]: Should we remove this? Seems like this doesn't affect any changes to the styling.

File: index.html [hunk 2]
```diff
@@ -102,6 +106,68 @@ <h3 class="card-title">Lorem, ipsum.</h3>
                 </div>
             </div>
         </div>
+        <div class="container mt-4">
+            <div class="sgds accordion" id="accordionExample">
+                <div class="accordion-item">
+                    <h2 class="accordion-header" id="headingOne">
+                        <button class="sgds accordion-button" type="button" data-bs-toggle="collapse"
```
Comment (rav3) [inline index.html:113]: may want to remove the sgds class here

File: index.html [hunk 3]
```diff
@@ -102,6 +106,68 @@ <h3 class="card-title">Lorem, ipsum.</h3>
...
+                            data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                            Accordion Item #1
+                        </button>
+                    </h2>
+                    <div id="collapseOne" class="sgds accordion-collapse collapse show" aria-labelledby="headingOne"
```
Comment (rav3) [inline index.html:118]: may want to remove the sgds class here

File: sgds/sass/sgds-theme/components/_accordion.scss (hunk 1 above)
Comment (clukhei) [reply sgds/sass/sgds-theme/components/_accordion.scss:3]: its changing the accordion-body line-height

**Pull request merged**